sources/arch/x86_64/pit.h

```diff
@@ -1,9 +1,12 @@
 #pragma once
 
 #define PIT_FREQUENCY 1193182
+#define PIT_MSEC 1000
```

Comment:
use micro or mili maybe ?

Reply:
wdym 
MS = Micro Second 
I think that MSEC good enough :thinking:

Reply:
US = Micro Second